pkgs/tools/security/fail2ban/default.nix

```diff
@@ -47,7 +47,12 @@ python3.pkgs.buildPythonApplication rec {
     in
     ''
       # see https://github.com/NixOS/nixpkgs/issues/4968
-      rm -rf ${sitePackages}/etc ${sitePackages}/usr
+      if [ -e "${sitePackages}/etc" ]; then
+        rm -r "${sitePackages}/etc"
+      fi
+      if [ -e "${sitePackages}/usr" ]; then
+        rm -r "${sitePackages}/usr"
+      fi
```

Comment:
I think I described myself wrong. I was more thinking about doing this conditional based on stdenv.isLinux/stdenv.isDarwin.

Reply:
I'll do that instead later today.

Reply:
@griff ping

Reply:
@SuperSandro2000 better?

Reply:
yeah, thank you very much